### PR TITLE
Add header link items mobile

### PR DESCRIPTION
**Pull request by helloimela**
**Describe pull-request**</br>
Added header item links in mobile view
Add fixed view with 100% height in mobile view

**Additional context**<br/>
Related PR  https://github.com/scania/corporate-ui-dev/pull/174

### CODE REVIEW

File: src/styles/c-navigation.scss [hunk 1]
```diff
@@ -78,8 +84,11 @@ ul {
 .navbar {
   display: block;
   padding: 0;
-  transition: margin 250ms ease;
+  transition: all 250ms ease;
   order: -1; /* This is needed for ie - due to it beeing render after slots */
+  height: calc(75vh - 64px);
```
Comment (awinny) [inline src/styles/c-navigation.scss]: Would be nice with a comment describing what the this calculation is about

File: src/styles/c-header.scss [hunk 2]
```diff
@@ -182,28 +241,22 @@ $link-color: #747472;
...
-      }
-    }
+}
 
-    ::slotted(c-navigation),
-    &::slotted(c-navigation) {
-      -webkit-box-shadow: 0px 17px 15px -15px rgba(0, 0, 0, 0.15);
-      box-shadow: 0px 17px 15px -15px rgba(0, 0, 0, 0.15);
+// IE special handling
+@media all and (-ms-high-contrast: none), (-ms-high-contrast: active) {
```
Comment (awinny) [inline src/styles/c-header.scss]: Try to set the position static as a general rule for all slotted a nodes in all browsers

File: src/styles/c-header.scss [hunk 3]
```diff
@@ -137,17 +179,16 @@ $link-color: #747472;
...
-    top: var(--stickyMargin);
-  }
+    // IE fallback
+    position: relative;
+    transition: all 0.2s ease-out;
 
-  ::slotted(c-navigation){
-    background-color: transparent;
-    border-top: 1px solid #e2e2e2;
+    * {
```
Comment (awinny) [inline src/styles/c-header.scss]: Do we really need this?

**Review by awinny — CHANGES_REQUESTED** [review]
See the line comments

File: src/styles/c-header.scss (hunk 3 above)
Comment (helloimela) [reply src/styles/c-header.scss]: Nope, I will remove this

File: src/styles/c-header.scss (hunk 2 above)
Comment (helloimela) [reply src/styles/c-header.scss]: Ok, I change some properties so we don't need this rule anymore

File: src/styles/c-navigation.scss (hunk 1 above)
Comment (helloimela) [reply src/styles/c-navigation.scss]: comment has been added in the new changes

**Review by awinny — APPROVED** [review]
Seems good 👍

**Pull request merged**